### PR TITLE
User Show and Index

### PR DESCRIPTION
**Pull request by SSalekin**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class UsersController < ApplicationController
+  before_action :load_user, only: [:show, :edit, :update]
```
Comment (noor-framgia) [inline app/controllers/users_controller.rb]: cancancan?

**Comment by noor-framgia** [comment]
ok

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def index
+    @users = User.all
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: no need

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module UsersHelper
+  def gravatar_for user, options = {size: 160}
```
Comment (dieunb) [inline app/helpers/users_helper.rb]: we have avatar uploader, so no need this.

**Pull request merged**